<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QSplitter" name="splitter3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>229</width>
              <height>282</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Draw</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Brush</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QToolButton" name="BrushtoolButton">
                     <property name="text">
                      <string>Brush</string>
                     </property>
                     <property name="icon">
                      <iconset resource="Image.qrc">
                       <normaloff>:/Image/draw2.png</normaloff>:/Image/draw2.png</iconset>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="spinBox"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Figure</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QToolButton" name="CircletoolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="Image.qrc">
                       <normaloff>:/Image/circle2.png</normaloff>:/Image/circle2.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="RectangletoolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="Image.qrc">
                       <normaloff>:/Image/triangle2.png</normaloff>:/Image/triangle2.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="TriangletoolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="Image.qrc">
                       <normaloff>:/Image/rectangle2.png</normaloff>:/Image/rectangle2.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="ArrowtoolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="Image.qrc">
                       <normaloff>:/Image/arrow3.png</normaloff>:/Image/arrow3.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Measure</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QToolButton" name="RulertoolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="Image.qrc">
                       <normaloff>:/Image/ruler.png</normaloff>:/Image/ruler.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="MeasuretoolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="Image.qrc">
                       <normaloff>:/Image/measure.png</normaloff>:/Image/measure.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="ProtractortoolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                     <property name="icon">
                      <iconset resource="Image.qrc">
                       <normaloff>:/Image/progtractor.png</normaloff>:/Image/progtractor.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QToolButton" name="ColortoolButton">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Color</string>
                 </property>
                 <property name="icon">
                  <iconset resource="Image.qrc">
                   <normaloff>:/Image/color.png</normaloff>:/Image/color.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>150</width>
                   <height>15</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>229</width>
              <height>282</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Image Processing</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Image</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <item>
                      <widget class="QToolButton" name="ColortoolButton_9">
                       <property name="text">
                        <string>...</string>
                       </property>
                       <property name="icon">
                        <iconset resource="Image.qrc">
                         <normaloff>:/Image/colorreversal.png</normaloff>:/Image/colorreversal.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="ColortoolButton_7">
                       <property name="text">
                        <string>...</string>
                       </property>
                       <property name="icon">
                        <iconset resource="Image.qrc">
                         <normaloff>:/Image/colorreversal.png</normaloff>:/Image/colorreversal.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="ColortoolButton_8">
                       <property name="text">
                        <string>...</string>
                       </property>
                       <property name="icon">
                        <iconset resource="Image.qrc">
                         <normaloff>:/Image/reversal.png</normaloff>:/Image/reversal.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_8">
                     <item>
                      <widget class="QToolButton" name="SharpentoolButton">
                       <property name="text">
                        <string>...</string>
                       </property>
                       <property name="icon">
                        <iconset resource="Image.qrc">
                         <normaloff>:/Image/sharpen.png</normaloff>:/Image/sharpen.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="ColortoolButton_10">
                       <property name="text">
                        <string>...</string>
                       </property>
                       <property name="icon">
                        <iconset resource="Image.qrc">
                         <normaloff>:/Image/bright.png</normaloff>:/Image/bright.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="ColortoolButton_12">
                       <property name="text">
                        <string>...</string>
                       </property>
                       <property name="icon">
                        <iconset resource="Image.qrc">
                         <normaloff>:/Image/panning.png</normaloff>:/Image/panning.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_9">
                     <item>
                      <widget class="QToolButton" name="ColortoolButton_6">
                       <property name="text">
                        <string>...</string>
                       </property>
                       <property name="icon">
                        <iconset resource="Image.qrc">
                         <normaloff>:/Image/rotate.png</normaloff>:/Image/rotate.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="ColortoolButton_11">
                       <property name="text">
                        <string>...</string>
                       </property>
                       <property name="icon">
                        <iconset resource="Image.qrc">
                         <normaloff>:/Image/zoom.png</normaloff>:/Image/zoom.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Initialize</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_10">
                   <item>
                    <widget class="QPushButton" name="pushButton_3">
                     <property name="text">
                      <string>Draw</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_4">
                     <property name="text">
                      <string>Image</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTableView" name="patientTableView">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>15</number>
        </attribute>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QMdiArea" name="mdiArea">
            <property name="viewMode">
             <enum>QMdiArea::TabbedView</enum>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTreeWidget" name="treeWidget">
               <column>
                <property name="text">
                 <string notr="true">1</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>18</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>18</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_2">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>18</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTreeWidget" name="treeWidget_2">
               <column>
                <property name="text">
                 <string notr="true">1</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>18</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_3">
                 <property name="text">
                  <string>Layout
Save</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>18</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_4">
                 <property name="text">
                  <string>Layout
Clear</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>18</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>12</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1248</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>저장</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections/>
</ui>
